sdk/cmake/CMakeMacros.cmake

```diff
@@ -577,8 +577,58 @@ list(APPEND VALID_MODULE_TYPES kernel kerneldll kernelmodedriver wdmdriver nativ
 list(APPEND KERNEL_MODULE_TYPES kernel kerneldll kernelmodedriver wdmdriver)
 list(APPEND NATIVE_MODULE_TYPES kernel kerneldll kernelmodedriver wdmdriver nativecui nativedll)
 
+function(add_ntddi_defines MODULE TARGET_VERSION)
```

Comment:
Please, change the function name to `set_ntddi_defines` to follow other functions' naming convention